generator/dependencies.json

```diff
@@ -0,0 +1,5 @@
+{
+  "bs-platform": "^4.0.0",
+	"reason-react": "^0.4.2",
```

Comment:
There is a small indentation issue

Reply:
Right. For some reason, my VSCode keeps falling back to tabs.